### ISSUE TITLE
ANSI support in stderr

### ISSUE THREAD

**Issue by salimmj** [marimo-team/marimo#2037]
### Description

Hi,

In some of the code I have that I run as a script I use the `logging` library to print info (with colors) like this:
```
logging.info(f"\033[1;32mDirectory created at {save_path}\033[0m")
```

I noticed this didn't work well in `marimo` and realized that it only renders ANSI on `stdout`. I made an easy fix by routing my logging to `stdout` instead of `stderr`. So this is not a big deal, was wondering why not support `stderr` as well.

### Suggested solution

Render ANSI for `stderr` as well.

### Additional context

Related to https://github.com/marimo-team/marimo/issues/707

**Comment by mscolnick** [comment]
hi, this should be very doable since we already do this with stdout

**Comment by mscolnick** [comment]
Closed by #2110
